command/node_meta_read.go

```diff
@@ -0,0 +1,142 @@
+package command
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/posener/complete"
+)
+
+type NodeMetaReadCommand struct {
+	Meta
+}
+
+func (c *NodeMetaReadCommand) Help() string {
+	helpText := `
+Usage: nomad node meta read [-json] [-node-id ...]
+
+	Read a node's metadata. This command only works on client agents. The node
+	status command can be used to retrieve node metadata from any agent.
+
+  Changes via the "node meta apply" subcommand are batched and may take up to
+  10 seconds to propagate to the servers and affect scheduling. This command
+  will always return the most recent node metadata while the "node status"
+  command can be used to view the metadata that is currently being used for
+  scheduling.
```

Comment:
It's hard to tell for sure in the PR reviewer but I think these two paragraphs aren't intended the same. The top one looks like it has tabs?